src/coreclr/tools/aot/ILCompiler.TypeSystem.ReadyToRun.Tests/MarshalUtilsTests.cs

```diff
@@ -0,0 +1,71 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
cc @jkoritzinsky, @MichalStrehovsky, I couldn't find MarshalUtils tests so I have added this. It may need to cover more cases?